testing/python/integration.py

```diff
@@ -164,7 +162,6 @@ def test_hello(inject_me):
         reprec.assertoutcome(passed=1)
 
     def test_unittest_mock_and_pypi_mock(self, testdir):
-        pytest.importorskip("unittest.mock")
         pytest.importorskip("mock", "1.0.1")
```

Comment:
Is there still a reason for anyone to use `mock` from PyPI?

Reply:
Yes: https://github.com/pytest-dev/pytest-mock/pull/54